conflowgen/posthoc_analyses/inbound_to_outbound_vehicle_capacity_utilization_analysis_report.py

```diff
@@ -93,6 +94,8 @@ def get_report_as_graph(self, **kwargs) -> object:
         plot_type = kwargs.get("plot_type", "both")
 
         vehicle_type, capacities = self._get_capacities_depending_on_vehicle_type(kwargs)
+        if len(capacities) == 0:
```

Comment:
Please check the data type. Is len really the correct check? And is it even necessary to check at all?